.changeset/bitter-colts-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+"ensadmin": minor
+---
+
+Remove dynamic params page path for explore ens names
```

Comment:
```suggestion
Transitioned explore name detail page to use query param instead of path
Build `URLSearchParams` from existing query string to enable use of `NameLink` inside `/name?name=` pages
```